src/opm/input/eclipse/Schedule/BCProp.cpp

```diff
@@ -104,37 +104,48 @@ BCProp::BCFace::BCFace(const DeckRecord& record) :
         temperature = T.getSIDouble(0);
     }
     MechBCValue mechbcvaluetmp;
+    bool all_ok = true;
     if (const auto& P = record.getItem<BCKEY::STRESSXX>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.stress[0] = P.getSIDouble(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::STRESSYY>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.stress[1] = P.getSIDouble(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::STRESSZZ>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.stress[2] = P.getSIDouble(0);
+        all_ok = false;
     }
     mechbcvaluetmp.stress[3] = 0;
     mechbcvaluetmp.stress[4] = 0;
     mechbcvaluetmp.stress[5] = 0;
     if (const auto& P = record.getItem<BCKEY::DISPX>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.disp[0] = P.getSIDouble(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::DISPY>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.disp[1] = P.getSIDouble(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::DISPZ>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.disp[2] = P.getSIDouble(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::FIXEDX>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.fixeddir[0] = P.get<int>(0);
+        all_ok = false;
     }
     if (const auto& P = record.getItem<BCKEY::FIXEDY>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.fixeddir[1] = P.get<int>(0);
     }
     if (const auto& P = record.getItem<BCKEY::FIXEDZ>(); ! P.defaultApplied(0)) {
         mechbcvaluetmp.fixeddir[2] = P.get<int>(0);
+        all_ok = false;
+    }
+    if(all_ok){
```

Comment:
I believe the logic here is now 180 degree wrong... The all_ok variable should be named all_defaulted, then it should be
```
if (!all_defaulted) {
    mechbcvalue = mechbcvaluetmp;
}
```
(please also add space around the parenthesis above)